web/spec/supabase.yml

```diff
@@ -423,9 +423,23 @@ pages:
   auth.update():
     $ref: '@supabase/gotrue-js."GoTrueClient".GoTrueClient.update'
     notes: |
-      It's generally better to store user data in a table inside your public schema (i.e. `public.users`). 
+      User email: email updates will send an email to both the user's current and new email with a confirmation link.
+      
+      User metadata: It's generally better to store user data in a table inside your public schema (i.e. `public.users`). 
       Use the `update()` method if you have data which rarely changes or is specific only to the logged in user.
     examples:
+      - name: Update email for authenticated user.
+        isSpotlight: true
+        js: |
+          ```js
+          const { user, error } = await supabase.auth.update({email: 'new@email.com'})
```

Comment:
@thorwebdev correct me if I'm wrong, but doesn't the payload need to be inside `data`?

```
update({data : { email: 'new@email.com' } })
```

Reply:
@kiwicopple it does not, `data` is only meant for metadata on the user, see https://github.com/supabase/gotrue-js/blob/master/src/lib/types.ts#L59-L77